### ISSUE TITLE
Some questions about human36m_diffpose_uvxyz_cpn.yml

### ISSUE THREAD

**Issue by jiejie22333** [GONGJIA0208/Diffpose#3]
I am new to 3D HPE，may I ask you some questions？
in **human36m_diffpose_uvxyz_cpn.yml**
data:
    dataset: "human36m"
    dataset_path: "./data/data_3d_h36m.npz"
    ##why use gt_gmm?##
    dataset_path_train_2d: "./data/data_2d_h36m_gt_gmm.npz"
    dataset_path_test_2d: "./data/data_2d_h36m_cpn_ft_h36m_dbb_gmm.npz"
    num_joints: 17
    num_workers: 32

why path_train uses **data_2d_h36m_gt_gmm.npz** and path_test uses **data_2d_h36m_cpn_ft_h36m_dbb_gmm.npz**

**Comment by GoogQing** [comment]
Human3,.6m has two settings: use the GT 2D pose or use the CPN detected 2D pose. 
During model training, we find that using the GT 2D pose (in the training set) to optimize our diffusion model can achieve better performance on both GT 2D pose (in the test set) and the CPN-detected 2D pose (in the test set). Thus, we use the GT pose for model training in the CPN pose setting.

**Comment by jiejie22333** [comment]
thank you ！！！

**Comment by jiejie22333** [comment]
May I ask when will you release your codes about Video-based experiments ?

**Comment by GoogQing** [comment]
Sorry for the delay. We are very busy in recent days due to several emergent projects. We will try to release the video-based code by the end of this month.

**Comment by xiu-cs** [comment]
> Human3,.6m has two settings: use the GT 2D pose or use the CPN detected 2D pose. During model training, we find that using the GT 2D pose (in the training set) to optimize our diffusion model can achieve better performance on both GT 2D pose (in the test set) and the CPN-detected 2D pose (in the test set). Thus, we use the GT pose for model training in the CPN pose setting.

Thanks for clarifying this. However, I think using GT 2D poses for training under the CPN-detected 2D pose setting is somewhat unfair.
